README.rst

```diff
@@ -0,0 +1,306 @@
+##########
+Quickstart
+##########
+
+.. _section-overview:
+
+Overview
+========
+
+
+Data Backbone (DBB) buffer manager runs on handoff site.
+
+.. _section-dbbbm:
+
+DBB buffer manager
+==================
+
+.. _section-dbbbm-download:
+
+Download deployment scripts
+---------------------------
+
+Recommended way of deploying DBB buffer manger is with Docker container.  You
+can download scripts which help you build and run such a container from
+GitHub:
+
+.. code-block:: bash
+
+   git clone https://github.com/lsst-dm/dbb_buffmngrs_deploy .
+   cd dbb_buffmngrs_deploy
+
+.. _section-dbbbm-configuration:
+
+Configure DBB buffer manager
+----------------------------
+
+In ``docker/dbb_buffmngrs_handoff/buffmngr.yml``, in **endpoint** section you
+need to specify three directories:  
+
+1. **staging area**: directory where the files are being transferred to,
+2. **storage area**: directory where the files will be moved after transfer is
+   complete,
+3. **buffer**: directory used by DBB ingestion service,
+
+as well as
+
+4. **host**: the name of the host acting as the endpoint site,
+5. **user**: a user account to use for file transfer (DBB buffer manger uses
+   internally ``scp`` to transfer files between the sites, so anonymous
+   transfer is not supported).
+
+.. warning::
+
+   The selected user needs to have passwordless login enabled on the endpoint
+   site.  
+   
+.. note:: 
+
+   Only *handoff* and *endpoint* sections are strictly required to be present
+   in the configuration file.  However, enabling logging to a file (by default
+   DBB buffer manager outputs log messages on stdout/stderr) is highly
+   recommended. You can achieve it by specifying **file** option in **logging**
+   section, for example:
+
+   .. code-block::
+   
+      logging:
+        file: /var/log/lsst/handoff.log
```

Comment:
Not sure if you want to include a note about choosing a logging dir to be a location mounted writeable in the docker (see bind section) in order to easily view logs outside of the docker.